config/src/config/mempool_config.rs

```diff
@@ -10,6 +10,7 @@ pub struct MempoolConfig {
     pub shared_mempool_tick_interval_ms: u64,
     pub shared_mempool_batch_size: usize,
     pub shared_mempool_max_concurrent_inbound_syncs: usize,
+    pub shared_mempool_min_broadcast_recipient_count: usize,
```

Comment:
can this be optional?
e.g. I wonder if at the end actually the best option will be to do nothing and rely on low ttl in mempool